support/metrics/metricsTracker.go

```diff
@@ -228,33 +257,37 @@ func (mt *MetricsTracker) SendUpdateEvent(now time.Time, success bool, millisFor
 
 // SendDeleteEvent sends the delete Amplitude event.
 func (mt *MetricsTracker) SendDeleteEvent(exit bool) error {
-	commonProps := mt.props
-	commonProps.SecondsSinceStart = time.Now().Sub(mt.botStartTime).Seconds()
+	mt.props.SecondsSinceStart = time.Now().Sub(mt.botStartTime).Seconds()
 	deleteProps := deleteProps{
-		commonProps: commonProps,
-		Exit:        exit,
-		StackTrace:  string(debug.Stack()),
+		Exit:       exit,
+		StackTrace: string(debug.Stack()),
 	}
 
-	return mt.sendEvent(deleteEventName, deleteProps)
+	return mt.SendEvent(deleteEventName, deleteProps)
 }
 
-func (mt *MetricsTracker) sendEvent(eventType string, eventProps interface{}) error {
+// SendEvent sends an event with its type and properties to
```

Comment:
this comment sentence seems incomplete.
let's finish the commend, or remove the "to"